docs/themes/oso-webpack/index.js

```diff
@@ -163,11 +206,136 @@ import('monaco-editor-core').then(monaco => {
     for (let i = 0; i < polarCode.length; i++) {
       let el = polarCode[i];
       monaco.editor
-        .colorize(el.innerText, 'polar', { theme: 'polarTheme' })
+        .colorize(el.innerText, 'polar', {
+          theme: 'polarTheme'
+        })
         .then(colored => {
           el.innerHTML = colored;
           el.parentNode.classList.add('polar-code-in-here');
         });
     }
   });
 });
+
+import('tinykeys').then(tinykeys => {
+  window.handleArrowDown = function(e) {
+    console.log("handleArrowDowns");
+  }
+
+  window.handleArrowUp = function(e) {
+    console.log("handleArrowUp");
+  }
+
+  window.hideSearch = function(e) {
+    if (searchModal.style.display == '') {
+      searchModal.style.display = 'none';
+    }
+  }
+
+  window.tinykeys(window, {
+    "ArrowDown": (e) => {
+      window.handleArrowDown(e);
+    },
+    "ArrowUp": (e) => {
+      window.handleArrowUp(e);
+    },
+    "Escape": (e) => {
+      window.hideSearch(e);
+    }
+  });
+});
+
+import('algoliasearch').then(_ => {
+  const Handlebars = require("handlebars");
+  const searchResult = require("./search-result.handlebars");
+  const searchInput = document.getElementById('search-input');
+  if (searchInput) {
+    searchInput.addEventListener('input', (e) =>
+      window.searchInputKeyUp(e)
+    );
+  }
+
+  const searchResultsContainer = document.getElementById('search-results-container');
+  const algoliasearch = require("algoliasearch");
+  const client = algoliasearch("KROZ8F05YT", "13594a3b7da482e011ce0ab08fdb4c4d");
```

Comment:
Are these keys safe to include in code? I realize we may need to migrate the Algolia account.

Reply:
Yes these are the read only ones necessary for JS